common/src/main/java/org/dst/entity/SortedListEntity.java

```diff
@@ -0,0 +1,34 @@
+package org.dst.entity;
+
+public class SortedListEntity implements Comparable<SortedListEntity> {
+  private int score;
```

Comment:
Please put all of the implementations of SortedList data structure to one package, such as `org.dst.xxx.yyy.sortedlist`